system-int-test/test/cmr/system_int_test/ingest/collection/collection_ingest_test.clj

```diff
@@ -792,7 +792,7 @@
   (testing "Ingest of a json collection with invalid mimetypes. This verifies that we validate mimetypes against kms."
     (let [coll-metadata (slurp (io/resource "CMR-7647/CMR-7647.json"))
           {:keys [status errors]} (ingest/ingest-concept
-                                   (ingest/concept :collection "PROV1" "foo" :umm-json coll-metadata)
+                                   (ingest/concept :collection "PROV1" "foo" {:format :umm-json :version "1.18.2"} coll-metadata)
```

Comment:
why 1.18.2 instead of 1.18.3?

Reply:
When a file gets ingested and the version isn't listed, the ingest code always assumes the last version.  But the example code MetadataSpecification lists the actual version of the record. 
In order to not update the test file every time we have an UMM-C change I add the version.  This was because this test failed because the data file is using an earlier version then the latest.